scripts/ci/detect_ci_change_areas.py

```
@@ -17,16 +17,18 @@ class ChangeAreas:
     macos: bool
     web: bool
     go: bool
+    agent_session_web: bool
 
     @classmethod
     def all(cls) -> "ChangeAreas":
```

Ruff (0.15.17)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)